### PR TITLE
created tests for bug-form actual results

### PR DESCRIPTION
**Pull request by ArtamonovDmytro**

### CODE REVIEW

File: test/specs/reg/bug-form/actual-result.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        help.login();
+        $(sel.newBug).click();
+        let element = $(sel.actualField);
+        let fieldFontFamily = element.getCSSProperty('font-family').value;
+        assert.equal(fieldFontFamily, exp.fieldFontFamily);
+    });
+
+    it('Font Size', function () {
+        help.login();
+        $(sel.newBug).click();
```
Comment (amadjouga) [inline test/specs/reg/bug-form/actual-result.js]: Tests seem good.
Not necessary to login and click new bug button in each test, you can do that once in first test is enough.
Tests will work faster.

File: test/expected/bug-form.js [hunk 2]
```diff
@@ -1,4 +1,10 @@
 export default {
+  fieldFontFamily: 'segoe ui',
+  fieldFontSize: '16px',
+  fieldFontWeight: 400,
```
Comment (amadjouga) [inline test/expected/bug-form.js]: You do duplicates for expected parameters, somebody already created them(like FontWeight, etc.)

File: test/expected/bug-form.js [hunk 3]
```diff
@@ -1,6 +1,10 @@
 export default {
+
+  fieldFontColor: '#495057',
+  fieldTextAlign: 'start',
+  fieldPlaceholderFontColor: '#6c757d',
   fontFamily: 'segoe ui',
-  fontSize: 16,
```
Comment (Miksyuk) [inline test/expected/bug-form.js]: are you sure that nobody used this FontSize in the tests made before you?

File: test/specs/reg/bug-form/actual-result.js (hunk 1 above)
Comment (Miksyuk) [reply test/specs/reg/bug-form/actual-result.js]: yes. help.login is not necessary in each test, only in 1

**Comment by artsenius** [comment]
Please fix the remaining issues and resolve conflicts

File: test/specs/reg/bug-form/actual-result.js (hunk 1 above)
Comment (artsenius) [reply test/specs/reg/bug-form/actual-result.js]: Still not fixed. You're clicking this button everytime. Just see: you're on the page. You can just go and check everything in one run, but instead you're clicking the button everytime.

File: test/specs/reg/bug-form/actual-result.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+    it('Font Size', function () {
+        $(sel.newBug).click();
+        let element = $(sel.actualResult);
+        let fontSize = element.getCSSProperty('font-size').value;
+        assert.equal(fontSize, exp.fontSize);
+    });
+
+    it('Font Weight', function () {
+        $(sel.newBug).click();
```
Comment (artsenius) [inline test/specs/reg/bug-form/actual-result.js]: remove also this

File: test/specs/reg/bug-form/actual-result.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        let element = $(sel.actualResult);
+        let fontWeight = element.getCSSProperty('font-Weight').value;
+        assert.equal(fontWeight, exp.fontWeight);
+    });
+
+    it('Font Color', function () {
+        $(sel.newBug).click();
```
Comment (artsenius) [inline test/specs/reg/bug-form/actual-result.js]: and this

File: test/specs/reg/bug-form/actual-result.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+        let element = $(sel.actualResult);
+        let fieldFontColor = element.getCSSProperty('color').parsed.hex;
+        assert.equal(fieldFontColor, exp.fieldFontColor);
+    });
+
+    it('text align', function () {
+        $(sel.newBug).click();
```
Comment (artsenius) [inline test/specs/reg/bug-form/actual-result.js]: and this

File: test/specs/reg/bug-form/actual-result.js [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+    it('Font Family', function () {
+        help.login();
+        $(sel.newBug).click();
+        let element = $(sel.actualResult);
+        let fontFamily = element.getCSSProperty('font-family').value;
+        assert.equal(fontFamily, exp.fontFamily);
+    });
+
+    it('Font Size', function () {
+        $(sel.newBug).click();
```
Comment (artsenius) [inline test/specs/reg/bug-form/actual-result.js]: Still no fix here

**Pull request merged**